ci/docker_ci.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -e
+
```

Comment:
is this chmod 755 ?

Reply:
e.g. see run_tests.sh for an example of the mode we need

Reply:
This line is identical to our other scripts. Not sure why it doesn't work.

Reply:
no it's the file itself i think
so chmod 755 ci/docker_.... and commit it